panel/panel.mjs

```diff
@@ -237,7 +242,7 @@ export default class CanjsDevtoolsPanel extends StacheElement {
 						vm.breakpointsError = detail;
 						break;
 					case "success": {
-						vm.componentTree.updateDeep(detail.tree || []);
+						Reflect.updateDeep(vm.componentTree, detail.tree || []);
```

Comment:
There must not be tests for this. Can we add some?

Reply:
Will do!